### DISCUSSION TITLE
Can i add local language for translation?

### DISCUSSION THREAD

**Discussion by the-wrong-guy** [benawad/dogehouse#1293] [Q&A] [answered]
I am thinking of adding my native language i.e. "Axomiya ( know as Assamese)", a language that is spoken in the Indian State of Assam, but if I add it, what flag and folder name should I should I use ?

**Comment by drvladb** [comment]
I don't think you need a flag / emoji, you can just leave it blank

**Comment by imadill** [comment, marked as answer]
> I am thinking of adding my native language i.e. "Axomiya ( know as Assamese)", a language that is spoken in the Indian State of Assam, but if I add it, what flag and folder name should I should I use ?

You need to name the folder according to ISO **639-1** code. Here are all the [codes](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes). 
Then you need to also add you language in this [file](https://github.com/benawad/dogehouse/blob/staging/kofta/src/app/components/LanguageSelector.tsx). You need to add your flag too (as Assam is in India, so it would be the Indian flag) :)

**Reply by the-wrong-guy** [reply]
well thanks
